app/routes/routes.js

```diff
@@ -1,6 +1,6 @@
 import { Router } from "../deps.js";
 import * as addressController from "./controllers/addressController.js";
-import * as userController from "./controllers/userController.js";
+//import * as userController from "./controllers/userController.js";
```

Comment:
Why did user controller commented?

Reply:
Its because  this line // router.post("/register", userController.createUser); already commented in the code when i pull it from Github.. Routes doesnt use the usercontroller so commented Import. Hope this clarifies.